lib/src/cli.dart

```diff
@@ -68,6 +68,7 @@ Future runLinter(List<String> args, LinterOptions initialLintOptions) async {
     ..addMultiOption('rules',
         help: 'A list of lint rules to run. For example: '
             'avoid_as,annotate_overrides')
+    // TODO(srawlins): Remove this flag; it does not work any more.
```

Comment:
As a first step, consider marking the option as hidden and updating the documentation to indicate that it is ignored.